src/sentry/web/urls.py

```diff
@@ -416,7 +415,7 @@ def init_all_applications():
     # Settings - Projects
     url(
         r'^(?P<organization_slug>[\w_-]+)/(?P<project_slug>[\w_-]+)/settings/$',
-        ProjectSettingsView.as_view(),
+        react_page_view,
```

Comment:
It seems we have two ways of doing this, link to absolute uri or define a route's view to `react_page_view` (and thus being able to keep using django's named routes). What's the preferred method here?